http/router/fs.lua

```diff
@@ -62,14 +64,19 @@ local function static_file(self, request, format)
         }
     end
 
-    local s, fh = pcall(io.input, file)
+    local fh, err = fio.open(file, {'O_RDONLY'})
 
-    if not s then
+    if err ~= nil then
         return { status = 404 }
     end
 
-    local body = fh:read('*a')
-    io.close(fh)
+    local body
+    body, err = fh:read()
+    if err ~= nil then
+        utils.errorf("Can not return static file for '%s': '%s'", request:path(), err)
```

Comment:
After we will return {status = 200, body = nil}. 200 - OK, but we have an error it's not OK.

Reply:
No. It's OK. After we will return error response with status = 500.

Reply:
Hm...

It's clear above when we `return { status = 404 }`, but completely unclear here

Reply:
But anyway it's better than it was before. I mean io to fio change